subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio.c

```diff
@@ -2511,6 +2511,15 @@ uint32_t radio_ccm_mic_is_valid(void)
 	return (NRF_CCM->MICSTATUS != 0);
 }
 
+void radio_ccm_disable(void)
+{
+	if (IS_ENABLED(CONFIG_SOC_SERIES_BSIM_NRFXX)) {
```

Comment:
given that as you explained in the PR description this is expected to also happen in real targets, shouldn't this be unconditional?